Change 27635 by stevep@stevep-mccoy on 2006/03/30 12:49:28

	Pod nit mentioned in Debian bug #358455.

Affected files ...

... //depot/perl/pod/perlsec.pod#37 edit

Differences ...

diff --exclude=debian -Naur perl-5.8.8.orig/pod/perlsec.pod perl-5.8.8/pod/perlsec.pod
--- perl-5.8.8.orig/pod/perlsec.pod	2004-12-31 06:20:41.000000000 +1100
+++ perl-5.8.8/pod/perlsec.pod	2006-04-05 00:06:41.000000000 +1000
@@ -497,7 +497,7 @@
 =back
 
 See L<http://www.cs.rice.edu/~scrosby/hash/> for more information,
-and any computer science text book on the algorithmic complexity.
+and any computer science textbook on the algorithmic complexity.
 
 =head1 SEE ALSO
 
